src/styles/bomshell.css

```diff
@@ -0,0 +1,208 @@
+               content: "";
+               display: block;
+               width: 14px;
+               height: 14px;
+               border-radius: 50%;
+               background-color: oklch(85.36% 0.1463 206.92);
+           }
+        }
+    }
+
+    button.custom-button {
+        height: 3rem;
+        width: 7rem;
+        margin-right: 2rem;
+        margin-top: 2rem;
+        border-radius: 0.25rem;
+        border: initial;
+        text-transform: uppercase;
+        background-color: initial;
+        &[data-variant="primary"] {
+            outline: 3px solid oklch(65.84% 0.2894 337.23);
+            background-image: radial-gradient(
+                farthest-side at top,
+                oklch(65.84% 0.2894 337.23 / 0.6),
+                oklch(65.84% 0.2894 337.23 / 0.2)
+            );
+        }
+        &[data-variant="secondary"] {
+            outline: 3px solid oklch(87.3% 0 0);
+            background-image: radial-gradient(
+                farthest-side at top,
+                oklch(87.3% 0 0 / 0.6),
+                oklch(87.3% 0 0 / 0.2)
+            );
+        }
+    }
+
+    sl-sidebar-state-persist > ul > li:has(> a) {
+        margin-top: initial;
+        &:first-child > a {
+            color: oklch(65.84% 0.2894 337.23);
+            background-color: initial;
+            border-radius: 4px;
+            outline: 2px solid oklch(65.84% 0.2894 337.23);
+            background-image: linear-gradient(to right, oklch(65.84% 0.2894 337.23 / 0.1), oklch(65.84% 0.2894 337.23 / 0.5));
+        }
+
+        &:first-child > a:is(:hover, :focus) {
+            background-image: radial-gradient(circle at top, oklch(65.84% 0.2894 337.23), oklch(49.84% 0.2894 337.23));
+            color: white;
+        }
+    }
+
+    sl-sidebar-state-persist ul > li > details > summary {
+        display: grid;
+        grid-auto-flow: column;
+        grid-template-columns: auto 1fr auto;
+        &::before {
+            content: "";
+            display: block;
+            grid-column: 2;
+            height: 2px;
+            background-image: linear-gradient(
+                to left,
+                oklch(35.19% 0.0207 269.43),
+                oklch(35.19% 0.0207 269.43 / 0)
+            );
+        }
+        &::after {
+            content: "";
+            grid-column: 3;
+            display: block;
+            width: 14px;
+            height: 14px;
+            background-color: oklch(35.19% 0.0207 269.43);
+        }
+    }
+
+    .caret {
+        display: none;
+    }
+
+    .group-label > .large, starlight-toc > nav > h2 {
+        color: oklch(87.3% 0 0);
+        text-transform: uppercase;
+    }
+
+    starlight-toc > nav > ul > li > a[aria-current="true"] {
+        color: white;
+    }
+
+    .sidebar-pane.sidebar-pane, .right-sidebar {
+        background-image: linear-gradient(
+            to bottom,
+            oklch(20% 0.0102 285.47) 0rem,
+            oklch(15% 0.0064 285.47) 26rem
+        );
+    }
```

Comment:
The fade effect for both the sidebars.

![image](https://github.com/user-attachments/assets/84678e4c-d068-47a4-b0a9-5ccc40d6848b)

![image](https://github.com/user-attachments/assets/bc08e463-2f07-4470-9720-05e0d0192216)